Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -87,6 +87,7 @@ LogManager::LogManager()
   m_Log[LogTypes::SP1] = new LogContainer("SP1", "Serial Port 1");
   m_Log[LogTypes::VIDEO] = new LogContainer("Video", "Video Backend");
   m_Log[LogTypes::VIDEOINTERFACE] = new LogContainer("VI", "Video Interface (VI)");
+  m_Log[LogTypes::VIDEOCUSTOMTEX] = new LogContainer("VCT", "Video Custom Textures (VCT)");
```

Comment:
I have asked for consultations with nobody responding I simply used the strings according to the existing pattern, so the initial changes shouldn't be surprising imo, for someone unfamiliar with the project.